packages/remix-dev/tsconfig.json

```diff
@@ -13,6 +13,7 @@
 
     "declaration": true,
     "emitDeclarationOnly": true,
+    "skipLibCheck": true,
```

Comment:
@jacob-ebey : what was the reason for enabling `skipLibCheck`? Is it because `esbuild-plugin-cache` pulls in `deno-cache` and `deno-importmap` as dependencies and type checking fails for those?